getting-started/scaling-out.md

```diff
@@ -1,17 +1,16 @@
 # Scaling out using Distributed Hypertables
 
->:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
-This method of deployment is not meant for production use. For more information, please
+>:WARNING: Distributed hypertables are currently in PRIVATE BETA and
+are not yet meant for production use. For more information, please
 [contact us][contact].
 
-TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives.
-Hypertables are used to handle large amount of data by breaking it up
-into smaller pieces (chunks), allowing operations to execute efficiently. When
-the amount of data is expected to be beyond what a single machine can handle,
-you can distribute the data chunks over several machines by using
-*distributed hypertables*.
-Distributed hypertables are similar to normal hypertables, but they
-add an additional layer of hypertable partitioning by distributing chunks
+Hypertables handle large amounts of data by breaking it up into
+smaller pieces (chunks), allowing operations to execute
+efficiently. When the amount of data is expected to be beyond what a
+single machine can handle, you can distribute the data chunks over
+several machines by using *distributed hypertables*.  Distributed
+hypertables are similar to normal hypertables, but they add an
+additional layer of hypertable partitioning by distributing chunks
```

Comment:
Suggestion: "... but they allow partitioning the hypertable further by distributing..."

Reply:
I am not too fond of this suggestion because I don't think distributed hypertables allow further partitioning. In fact, you can partition the same way on a single node and the only difference is where chunks are stored.

Also, FWIW, this is not new content, just updated because of formatting (line breaks).